Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/nhin/InboundDocRetrieveStrategyImpl.java

```diff
@@ -103,63 +112,54 @@ public void execute(InboundDocRetrieveOrchestratable message) {
      */
     public RetrieveDocumentSetResponseType sendToAdapter(InboundDocRetrieveOrchestratable message) {
         RetrieveDocumentSetResponseType adapterResponse = proxy.retrieveDocumentSet(message.getRequest(),
-                message.getAssertion());
+            message.getAssertion());
 
         try {
             DocRetrieveFileUtils.getInstance().convertFileLocationToDataIfEnabled(adapterResponse);
         } catch (Exception e) {
             LOG.error("Failed to retrieve data from the file uri in the payload.", e);
```

Comment:
I know this isn't something you wrote, but as long as you're in this class and we're on topic, can you also fix the way this exception is logged?